src/utils/internalLinks.js

```diff
@@ -217,7 +231,7 @@ export const getNearOperandFromLink = (url: URL, realm: URL): number | null => {
       i % 2 === 0
       // The operator is 'near' and its meaning is not negated (`str` does
       // not start with "-").
-      && str === 'near',
+      && str === (actuallyWith ? 'with' : 'near'),
```

Comment:
How about:
```suggestion
      && (str === 'near' || str == 'with'),
```

I feel like that'd be somewhat less of a hack — it means we're treating `with` as a synonym of `near`, which is exactly what we intend.

Reply:
Would this be a good place for this quote—

```
*   […] Quoting Greg from #5866:
*   > Currently the app doesn't interpret the "anchor" meaning of `/near/`
*   > links at all (that's #3604) — we already effectively give `/near/`
*   > links exactly the meaning that the upcoming `/with/` links will have.
*   > So to handle the new links, we just need to parse them and then
*   > handle them the way we already handle `/near/` links.
```

? Treating `with` as a synonym of `near` is a hack, and this quote helps communicate why that's basically OK for zulip-mobile.